src/components/rangeslider/index.js

```diff
@@ -0,0 +1,292 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Plotly = require('../../plotly');
+
+var Lib = require('../../lib');
+var Plots = require('../../plots/plots');
+
+var helpers = require('./helpers');
+var rangePlot = require('./range_plot');
+var attributes = require('./attributes');
+
+var svgNS = 'http://www.w3.org/2000/svg';
```

Comment:
let's require this guy in https://github.com/plotly/plotly.js/blob/master/src/constants/xmlns_namespaces.js